lib/ansible/plugins/filter/core.py

```diff
@@ -45,7 +45,7 @@
 from ansible.module_utils.six.moves import reduce, shlex_quote
 from ansible.module_utils._text import to_bytes, to_native, to_text
 from ansible.module_utils.common.collections import is_sequence
-from ansible.module_utils.common._collections_compat import Mapping, MutableMapping
+from ansible.module_utils.common._collections_compat import Mapping, MutableMapping, MutableSequence
```

Comment:
I don't see where in the code you're using this.

Reply:
Indeed, must be a left over from my tests/development.
I also removed MutableMapping which wasn't used either.